src/store/middlewares/account.js

```diff
@@ -15,6 +16,14 @@ const updateAccountData = next => (store) => { // eslint-disable-line
         confirmed: response.transactions,
         count: parseInt(response.count, 10),
       })));
+      if (account.isDelegate) {
+        store.dispatch(fetchAndUpdateForgedBlocks({
```

Comment:
This action returns a function that accepts dispatch as argument. Prefer using it inside the component when account property is updated and shows a change in balance.